jme3-examples/src/main/java/jme3test/water/TestSceneWater.java

```diff
@@ -51,7 +51,7 @@
 public class TestSceneWater extends SimpleApplication {
 
     // set default for applets
-    private static boolean useHttp = true;
```

Comment:
However this change is interesting.
Did all other tests have http = false? Also the comment about applets is outdated and probably inappropriate anyway, so probably good to set this to false.

Actually I guess Applets are the reason we had http/zip locators.

Reply:
Other tests have useHttp = false
It's the cause of that change.

Reply:
`TestQ3` has `useHttp = false`.
`TestMultipleFilters` has `useHttp = false`.
`TestSceneLoading` has `useHttp = false`.
`TestPostWaterLake` has `useHttp = false`.

So the answer is yes, all the other tests affected by this PR had `useHttp = false`.